<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditname">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>61</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>329</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prescribe">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开药</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>329</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_insert">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>329</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>插入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_searchall">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modify">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>1281</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="checkaddButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认插入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkaddButton_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>770</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认插入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkaddButton_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>780</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认插入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkaddButton_refresh">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>病人管理系统</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditid</tabstop>
  <tabstop>lineEditname</tabstop>
  <tabstop>pushButton_search</tabstop>
  <tabstop>pushButton_insert</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_searchall</tabstop>
  <tabstop>pushButton_modify</tabstop>
  <tabstop>pushButton_prescribe</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
